ui/app/mainui/AppMain.qml

```diff
@@ -724,6 +726,14 @@ Item {
                     closeBtnVisible: false
                 }
 
+                ModuleWarning {
+                    id: mailserverConnectionBanner
+                    type: ModuleWarning.Danger
+                    text: qsTr("Can not connect to mailserver. Retrying automatically")
```

Comment:
We no longer call it mailserver to users
```suggestion
                    text: qsTr("Can not connect to store node. Retrying automatically")
```

Reply:
fixed